golang/README.rst

```diff
@@ -0,0 +1,50 @@
+===========
+Golang Backend
+===========
```

Comment:
reStructuredText is picky. Those lines have to be aligned with the length of the section name.

```suggestion
==============
Golang Backend
==============
```